src/site/content/en/lang.11tydata.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
```

Comment:
Does this file work? Eleventy's docs say:

![image](https://user-images.githubusercontent.com/1066253/87724245-f42b5800-c76f-11ea-893c-912f2965824c.png)

https://www.11ty.dev/docs/data-template-dir/

Reply:
I opened an issue about it a while back https://github.com/11ty/eleventy/issues/492

Reply:
I don't think it'll trickle to other files, but seeing how it's being required in the blog it should be fine.

Reply:
Actually it does not need to be an 11tydata file, normal .js would do too.